browser/src/canvas/sections/CommentSection.ts

```diff
@@ -1192,6 +1181,10 @@ export class Comment extends CanvasSectionObject {
 		return null;
 	}
 
+	public static isAnyFocus(): boolean {
+		return document.activeElement.id.startsWith('annotation-modify-textarea') || document.activeElement.id.startsWith('annotation-reply-textarea');
```

Comment:
This is fragile, if someone will change id - we will have a bug.
Can we bind that by some shared variable or method, with the place where we create textarea?

Reply:
Thanks for checking. It should be safer now.